src/Native/Bootstrap/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ add_compile_options(-Wno-unused-private-field)
 add_compile_options(-Wno-tautological-undefined-compare)
 endif()
 
+if(WIN32)
+# assume that functions declared as extern "C" may throw an exception
```

Comment:
Do we have a similar problem when compiling the runtime? It is compiled with EHsc now.

Reply:
This was to fix compiler warnings since bootstrapper throws from extern C methods we use as workaround. I don't see warnings when compiling the runtime for some reason.

(Now that I think about it - suspiciously, I haven't seen these warnings when building the CppCodegen flavor of the bootstrapper which has had the same pattern for a while.)

Reply:
That's because of the `#pragma warning(disable:4297)` here: https://github.com/dotnet/corert/blob/master/src/Native/Bootstrap/main.cpp#L91

Move this pragma outside the CPPCODEGEN ifdef so that we have just one workaround for the same problem?

Reply:
Ah, missed that. Fixed, thanks!